crowdsourcer/templates/crowdsourcer/authority_questions_with_previous.html

```diff
@@ -0,0 +1,197 @@
+                </details>
+            </div>
+            <div class="row gx-lg-5">
+                <div class="col-lg-6 mt-6">
+
+                    <h3 class="h5 text-muted mb-3 mb-lg-4">Previous Response</h3>
+
+                    <h4 class="form-label fs-6">Marker’s answer</h4>
+                    <div class="read-only-answer mb-3 mb-md-4">
+                      {% if q_form.previous_response.multi_option.values %}
+                        <p>
+                          {% for option in q_form.previous_response.multi_option.values %}
+                            {{ option.description }},
+                          {% empty %}
+                            (none)
+                          {% endfor %}
+                        </p>
+                      {% else %}
+                        {{ q_form.previous_response.option|default:"(none)"|linebreaks }}
+                      {% endif %}
+                    </div>
+
+                  {% if q_form.question_obj.how_marked == 'foi' %}
+                    <h4 class="form-label fs-6">FOI request</h4>
+                    <div class="read-only-answer mb-3 mb-md-4">
+                        {{ q_form.previous_response.evidence|urlize_external }}
+                    </div>
+                  {% else %}
+                    <h4 class="form-label fs-6">Marker’s evidence of criteria met</h4>
+                    <div class="read-only-answer mb-3 mb-md-4">
+                        {{ q_form.previous_response.evidence|default:"(none)"|linebreaks }}
+                    </div>
+                  {% endif %}
+
+                  {% if q_form.question_obj.how_marked != 'foi' %}
+                    <h4 class="form-label fs-6">Links to evidence</h4>
+                    <div class="read-only-answer mb-3 mb-md-4">
+                        {{ q_form.previous_response.public_notes|default:"(none)"|urlize_external|linebreaks }}
+                    </div>
+
+                    <h4 class="form-label fs-6">Page number</h4>
+                    <div class="read-only-answer mb-3 mb-md-4">
+                        {{ q_form.previous_response.page_number|default:"(none)" }}<br>
+                    </div>
+                  {% endif %}
+
+                    <h4 class="form-label fs-6">Marker’s additional notes</h4>
+                    <div class="read-only-answer mb-3 mb-md-4">
+                        {{ q_form.previous_response.private_notes|default:"(none)"|linebreaks }}
+                    </div>
+
+                </div>
+                <div class="col-lg-6 mt-6 js-new-responses">
+
+                    <h3 class="h5 text-muted mb-3 mb-lg-4">New response</h3>
+
+                    <div class="mb-3">
+                        <button class="btn btn-sm btn-outline-primary d-flex align-items-center js-copy-response-from-previous" type="button">
+                          <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="me-2" viewBox="0 0 16 16">
```

Comment:
Now we’re using this "magic wand" icon in a second place, perhaps it’s time we moved it into its own partial, in `crowdsourcer/templates/crowdsourcer/icons/`? You can use `download.svg` as an example of how to handle the overrideable class/height/width attributes etc.